perf/intkey_workload/packaging/ubuntu/control

```diff
@@ -0,0 +1,11 @@
+Source: intkey_workload
+Maintainer: Hyperledger Sawtooth <unknown@unknown>
+Section: rust
+Priority: optional
+Standards-Version: 3.9.6
+Homepage: https://github.com/hyperledger/sawtooth-core
+Package: intkey-workload
+Architecture: all
```

Comment:
does this run on any architecture? If not this should be amd64.

Reply:
Ah, thanks. I think it should be amd64.